docs/graphql/manual/remote-schemas/schema-auth.rst

```diff
@@ -0,0 +1,103 @@
+.. meta::
+   :description: Authorization in your remote schema server with Hasura
+   :keywords: hasura, docs, remote schema, authorization
+
+.. _schema_auth:
+
+Authorization in your remote schema server
+==========================================
+
+.. contents:: Table of contents
+  :backlinks: none
+  :depth: 2
+  :local:
+
+Introduction
+------------
+
+Hasura will forward the resolved ``x-hasura-*`` values as headers to your remote
+schema. You can use this information to apply authorization rules in your
+server. You don't have to redo authentication in your remote schema server.
+
```

Comment:
Would be nice to put native fine grained access control for remote schemas will be released soon.

Reply:
This was existing copy but I'll update :)

Reply:
Oops nm new copy

Reply:
Ah yes, this was existing copy. I've added this sentence.